### PR TITLE
mainly bugfixing and a few extra things

### PR DESCRIPTION
**Pull request by hRobert12**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/hRobert12/My-games/pull/15%23issuecomment-204857729%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/15%23discussion_r58303574%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/15%23discussion_r58303575%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/15%23issuecomment-204859819%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/15%23discussion_r58314753%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hRobert12/My-games/pull/15%23issuecomment-204857729%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ca%20href%3D%5C%22https%3A//app.review.ninja/hRobert12/My-games/pull/15%5C%22%20target%3D%5C%22_blank%5C%22%3E%3Cimg%20src%3D%5C%22https%3A//app.review.ninja/46381900/pull/15/badge%5C%22%20alt%3D%5C%22ReviewNinja%5C%22/%3E%22%2C%20%22created_at%22%3A%20%222016-04-03T02%3A41%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8714728%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/review-ninja%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5Cn%5Cn%5Cn%5Cn%5Cn%5Cn%5Cn%5Cn%2A%5Brandom_game.py%2C%20line%2056%20%5C%5C%5Br2%5C%5C%5D%5D%28https%3A//reviewable.io%3A443/reviews/hrobert12/my-games/15%23-KEP42iZnSmPTnUPGXW9%3A-KEP42i_VPFcDNBmhu5a%3A1298947346%29%20%28%5Braw%20file%5D%28https%3A//github.com/hrobert12/my-games/blob/31a908eadd30d3bd7d98bd7c5c3475eb65e1258b/random_game.py%23L56%29%29%3A%2A%5Cnself-updater%3F%5Cn%5Cn---%5Cn%5Cn%5Cn%2AComments%20from%20%5BReviewable%5D%28https%3A//reviewable.io%3A443/reviews/hrobert12/my-games/15%29%2A%5Cn%3C%21--%20Sent%20from%20Reviewable.io%20--%3E%5Cn%22%2C%20%22created_at%22%3A%20%222016-04-03T03%3A07%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2031a908eadd30d3bd7d98bd7c5c3475eb65e1258b%20random_game.py%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hRobert12/My-games/pull/15%23discussion_r58303574%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Self-updater%3F%22%2C%20%22created_at%22%3A%20%222016-04-03T02%3A46%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%21resolve%22%2C%20%22created_at%22%3A%20%222016-04-03T02%3A46%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22postpone%20%21resolved%22%2C%20%22created_at%22%3A%20%222016-04-03T18%3A16%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20random_game.py%3AL55-101%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/hRobert12/My-games/pull/15#issuecomment-204857729'>General Comment</a></b>
- <a href='https://github.com/review-ninja'></a> <a href="https://app.review.ninja/hRobert12/My-games/pull/15" target="_blank">ReviewNinja
- <a href='https://github.com/hRobert12'></a> _[random_game.py, line 56 [r2]](https://reviewable.io:443/reviews/hrobert12/my-games/15#-KEP42iZnSmPTnUPGXW9:-KEP42i_VPFcDNBmhu5a:1298947346) ([raw file](https://github.com/hrobert12/my-games/blob/31a908eadd30d3bd7d98bd7c5c3475eb65e1258b/random_game.py#L56)):_
  self-updater?
  ---
  _Comments from [Reviewable](https://reviewable.io:443/reviews/hrobert12/my-games/15)_
  
- [ ] <a href='#crh-comment-Pull 31a908eadd30d3bd7d98bd7c5c3475eb65e1258b random_game.py 10'></a> &nbsp;<b><a href='https://github.com/hRobert12/My-games/pull/15#discussion_r58303574'>File: random_game.py:L55-101</a></b>
- <a href='https://github.com/hRobert12'></a> Self-updater?
- <a href='https://github.com/hRobert12'></a> !resolve
- <a href='https://github.com/hRobert12'></a> postpone !resolved

<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/15?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/15?approve=1'></a>&nbsp;<a href='https://github.com/hRobert12/My-games/pull/15'></a>
<a href='#crh-end'></a>

This change is [Reviewable](https://reviewable.io/reviews/hrobert12/my-games/15)

---

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/hRobert12/My-games/pull/15" target="_blank">ReviewNinja

File: random_game.py [hunk 1]
```diff
@@ -55,41 +55,46 @@ def checknumber(number):
```
Comment (hRobert12) [inline random_game.py:55]: Self-updater?

Comment (hRobert12) [inline random_game.py:55]: !resolve

**Comment by hRobert12** [comment]
_[random_game.py, line 56 [r2]](https://reviewable.io:443/reviews/hrobert12/my-games/15#-KEP42iZnSmPTnUPGXW9:-KEP42i_VPFcDNBmhu5a:1298947346) ([raw file](https://github.com/hrobert12/my-games/blob/31a908eadd30d3bd7d98bd7c5c3475eb65e1258b/random_game.py#L56)):_
self-updater?

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/hrobert12/my-games/15)_

File: random_game.py (hunk 1 above)
Comment (hRobert12) [inline random_game.py:55]: postpone !resolved

**Pull request merged**